ush/load_modules_run_task.sh

```diff
@@ -184,10 +184,10 @@ module list
 if [ -n "${SRW_ENV:-}" ] ; then
   set +u
   conda activate ${SRW_ENV}
-  if [ $machine = "gaea" ]; then
+#  if [ $machine = "gaea" ]; then
      conda deactivate
      conda activate ${SRW_ENV}
-  fi
+#  fi
```

Comment:
It looks like now you had to do the `deactivate->activate` process for all systems now -- we were hoping for the opposite with your PR! I guess it must be the Lua miniconda3 file epic installations that is causing this.

Reply:
@natalie-perlin Looking at PR #419, I didn't see this modification.  Is this required due to only using the new miniconda3 modules?  @mark-a-potts has opened PR #428 to remove this logic, along with the `conda deactivate` and `conda activate ${SRW_ENV}`.  It would be best to remove this, rather than make all machines require it.  Would you be able to run tests of removing this logic and making similar changes that @mark-a-potts made for the Gaea local modulefiles in PR #428?

Reply:
@danielabdi-noaa - this fix was temporary that we found would work on all systems. It is likely related to additional loads of miniconda3 in *.local.lua files if/when they are used. This module loading changes the binaries search $PATH once again. The use of *local.lua files may still need revisiting, checking the options now (@mark-a-potts does testing as well)